tests/snappi_tests/multidut/pfcwd/files/pfcwd_multidut_basic_helper.py

```diff
@@ -140,11 +141,11 @@ def run_pfcwd_basic_test(api,
                       warm_up_traffic_delay_sec, flow1_delay_sec, flow2_delay_sec],
                   data_flow_dur_sec_list=[
                       warm_up_traffic_dur_sec, flow1_dur_sec, flow2_dur_sec],
-                  data_pkt_size=DATA_PKT_SIZE,
+                  data_pkt_size=8192 if ingress_duthost.facts['asic_type'] == "cisco-8000" else 1024,
```

Comment:
why 1024 packet size doesn't work? I would prefer to leave this change in a separate PR.

Reply:
For long-link+HBM, Alpesh recommended to use 8K packet size, since that is the buffer size. I have not verified it on other platforms, but my deduction is it should not affect all other cisco platforms also.

Reply:
can you remove this last commit and put in another PR? as it's cisco platform specific.  Then, we can merge in the rest of the change first.    We can continue discuss the platform limitation in the new PR.

@abdosi for viz too.

Reply:
Done. Will raise a different PR for the cisco-8000 changes.